handbook/company/communications.md

```diff
@@ -565,7 +565,17 @@ The images used in the docs live in `docs/images/`. Note that you must provide t
 
 > Note that the instructions above also apply to adding images in the Fleet handbook.
 
+### Audit logs
 
+The [Audit logs doc page](../../docs/Using%20Fleet/Audit-logs.md) has a page generator that is used to speed up doc writing when Fleet adds new activity types.
```

Comment:
```suggestion
The [Audit logs doc page](https://fleetdm.com/docs/Using%20Fleet/Audit-logs.md) has a page generator that is used to speed up doc writing when Fleet adds new activity types.
```

^not sure that relative path links work across the handbook and docs like this, so I'd err on the side of using the `fleetdm.com/docs/…` url